e2e-tests/src/components/ProminentImage.tsx

```diff
@@ -0,0 +1,20 @@
+export interface ProminentImageProps {
+  src: string;
+  altText?: string;
+}
+
+export const initialProps: ProminentImageProps = {
+  src: "https://tellurideadventures.com/wp-content/uploads/2021/01/telluride-4100097_640.jpg",
+};
+
+export default function ProminentImage({ src, altText }: ProminentImageProps) {
```

Comment:
Do we want to remove this component if we're no longer using it?

Reply:
We'll be using this Component very soon.